tox.ini

```diff
@@ -20,6 +20,12 @@ commands =
     codespell dandi setup.py
     flake8 --config=setup.cfg {posargs} dandi setup.py
 
+[testenv:docs]
+basepython = python3
+deps = -rdocs/requirements.txt
+changedir = docs
+commands = sphinx-build -E -b html source build
```

Comment:
```suggestion
commands = sphinx-build -E -W -b html source build
```
so warnings are treated as errors? (if so -- please also add to `docs/Makefile` SPHINXOPTS) . unfortunately it would then error out upon encountering the first one, 

<details>
<summary>ATM I still observe some warnings to be dealt with</summary> 

```shell
$> python3 `which sphinx-build` -E -W -b html source build                                              
Running Sphinx v3.4.3
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 16 source files that are out of date
updating environment: [new config] 16 added, 0 changed, 0 removed
reading sources... [100%] modref/utils                                                                                                                                                                                                                             

Warning, treated as error:
/home/yoh/proj/dandi/dandi-cli-master/docs/source/cmdline/organize.rst:41:Unknown directive type "options".

.. options:: -d, --dandiset-path <dir>

    A top directory (local) of the dandiset to organize files under.  If not
    specified, dandiset current directory is under is assumed.  For 'simulate'
    mode target dandiset/directory must not exist.
(dev3) 1 22202 ->2.....................................:Fri 16 Jul 2021 12:44:05 PM EDT:.
(git)lena:~/proj/dandi/dandi-cli-master[enh-sphinx]docs
$> python3 `which sphinx-build` -E  -b html source build 
Running Sphinx v3.4.3
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 16 source files that are out of date
updating environment: [new config] 16 added, 0 changed, 0 removed
reading sources... [100%] modref/utils                                                                                                                                                                                                                             
/home/yoh/proj/dandi/dandi-cli-master/docs/source/cmdline/organize.rst:41: WARNING: Unknown directive type "options".

.. options:: -d, --dandiset-path <dir>

    A top directory (local) of the dandiset to organize files under.  If not
    specified, dandiset current directory is under is assumed.  For 'simulate'
    mode target dandiset/directory must not exist.
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiapi.py:docstring of dandi.dandiapi.RemoteDandiset.iter_upload_raw_asset:6: WARNING: Unexpected section title.

Parameters
----------
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiapi.py:docstring of dandi.dandiapi.RemoteDandiset.iter_upload_raw_asset:17: WARNING: Unexpected section title.

Returns
-------
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiapi.py:docstring of dandi.dandiapi.RemoteDandiset.upload_raw_asset:6: WARNING: Unexpected section title.

Parameters
----------
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiarchive.py:docstring of dandi.dandiarchive.navigate_url:4: WARNING: Unexpected section title.

Parameters
----------
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiarchive.py:docstring of dandi.dandiarchive.navigate_url:9: WARNING: Unexpected section title.

Yields
------
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiarchive.py:docstring of dandi.dandiarchive._dandi_url_parser.parse:5: WARNING: Unexpected indentation.
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiarchive.py:docstring of dandi.dandiarchive._dandi_url_parser.parse:16: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/yoh/proj/dandi/dandi-cli-master/dandi/dandiarchive.py:docstring of dandi.dandiarchive._dandi_url_parser.parse:20: WARNING: Unexpected section title.

Returns
-------
/home/yoh/proj/dandi/dandi-cli-master/docs/source/modref/index.rst.rst:44: WARNING: autosummary: stub file not found 'dandi.tests.fixtures'. Check your autosummary_generate setting.
/home/yoh/proj/dandi/dandi-cli-master/docs/source/modref/index.rst.rst:44: WARNING: autosummary: stub file not found 'dandi.tests.skip'. Check your autosummary_generate setting.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] modref/utils                                                                                                                                                                                                                              
/home/yoh/proj/dandi/dandi-cli-master/docs/source/cmdline/upload.rst:10: WARNING: unknown option: --dandiset-path
generating indices... genindex py-modindex done
writing additional pages... search done
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 12 warnings.

The HTML pages are in build.
```
</details>

Reply:
Done & fixed.